.gitignore

```diff
@@ -25,6 +25,7 @@ build/Release
 
 # Dependency directory
 node_modules
+package-lock.json
```

Comment:
Hi @ivanbtrujillo, thanks for opening this PR.  Is there a need to add this to our `.gitignore` and if not, can you update your change to not include this?

Reply:
The reason is that in the previous comment @johnstem ask for remove package-lock.json. 

I've read that is recommended to keep it  on the repo but I don't know the reasons that maintainers have to remove it.

Should I keep it or not? 

Thanks!

Reply:
If we are removing the file we should remove this line from the `.gitignore` as well.  

If you get that change committed and make sure the linter and test suite are passing I can run this again our projects and get it merged.